### PR TITLE
add field currentVersion for version conflict error, close #1002

### LINKED ISSUE #1002: add field currentVersion for version conflict error

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz please review

### CODE REVIEW

**Comment by schleichardt** [comment]
release note:

```html
 <li class=new-in-release>{@link ConcurrentModificationException#getCurrentVersion()} 
and {@link io.sphere.sdk.models.errors.ConcurrentModificationError} 
to get the current version number when a version conflict occurs</li>
```

File: commercetools-sdk-base/src/main/java/io/sphere/sdk/client/ConcurrentModificationException.java [hunk 1]
```diff
@@ -13,14 +21,53 @@
...
+        errors = Collections.emptyList();
+    }
+
+    public ConcurrentModificationException(final ErrorResponse errorResponse) {
+        this(errorResponse.getStatusCode(), errorResponse.getMessage(), errorResponse.getErrors());
+    }
+
+    ConcurrentModificationException(final Integer statusCode, final String message, final List<? extends SphereError> errors) {
+        super(message, statusCode);
+        System.err.println(errors);
```
Comment (lauraluiz) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/ConcurrentModificationException.java]: serr?

**Comment by lauraluiz** [comment]
:+1:

File: commercetools-sdk-base/src/main/java/io/sphere/sdk/client/ConcurrentModificationException.java (hunk 1 above)
Comment (schleichardt) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/ConcurrentModificationException.java]: Yes, I wasn't sure the debugger works. :)
I will remove it.

**Pull request merged**